src/releasers/go-yoshi.ts

```diff
@@ -0,0 +1,151 @@
+// * we need a deny list of modules that are mono repos, which we keep track of
+// ideally with minimal human intervention:
+//   * these should not show up in CHANGELOG for an "all" release.
+// * We need to also be able to manage the release of our snowflake, hand-written
+// libraries. Which get released.
+// * top level changes relate to all.
+
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {ReleasePR, ReleaseCandidate} from '../release-pr';
+
+import {ConventionalCommits} from '../conventional-commits';
+import {GitHubTag} from '../github';
+import {checkpoint, CheckpointType} from '../util/checkpoint';
+import {Update} from '../updaters/update';
+import {Commit} from '../graphql-to-commits';
+
+import {ReleaseType} from 'semver';
+import * as semver from 'semver';
+
+// Generic
+import {Changelog} from '../updaters/changelog';
+
+// These repos should be ignored when generating a release for the
+// top level mono-repo:
+const IGNORED_PATHS = [
```

Comment:
I'm ok with this for now, but ...  I do wonder out loud what a more sustainable model for `release-please` looks like.  I like the idea of built-in support for monorepo setups, and for split repo setups across languages.  For cases where there is very customer configuration like this, I'd very much like to start thinking about a config format you could use so this code can remain more generalized.  This isn't blocking feedback for this PR, just a seed to plant.

Reply:
I think we will eventually want this to be programatic, especially if we ever start split out into more and more sub-modules. But for now, this "works" for the top level module at least. I am still grokking what this code has context to. I have some ideas in the back of my head how we could make this better in the future. For now... 🤔

Reply:
so, we do have some support for mono-repos now:

https://github.com/googleapis/release-please/pull/501

But I was thinking this logic is a little yoshi  specific...

What I've been thinking  would be nice, would be to keep these yoshi  specific strategies  somewhat hidden, i.e.,  not calling them out  in documentation. And, when we add a strategy like yoshi-java, or yoshi-go, it would be nice to also add corresponding strategies that are closer to something the community could use.